lib/main.js

```diff
@@ -40,11 +41,22 @@ function shim(Linter) {
   return LinterShim;
 }
 
+/**
+ * Checks whether or not the given grammar is for TypeScript
+ * @param {Grammar} grammar The grammar to check
+ * @return {boolean} Whether or not the grammar is for TypeScript
+ */
+function isTypescriptGrammar(grammar) {
+  return grammar.fileTypes.find(fileType => /.*ts/i.test(fileType));
```

Comment:
Please put a note that this is not part of the public API if we can't find a way around using this.